update_dependency_licenses.py

```diff
@@ -54,27 +70,53 @@ def GetPackageLicenses(package_name):
         else:
             match = re.search(r"^License:\s*(.*)$", metadata,
                               re.MULTILINE | re.IGNORECASE)
+            assert match
             package_license = re.sub(r"\sLicences|\sLicense,?|Version\s", "",
                                      match.group(1))
             package_license = re.sub("-", " ", package_license)
         match = re.search(r"^Home-page:\s*(.*)$", metadata,
                           re.MULTILINE | re.IGNORECASE)
+        assert match
         homepage = match.group(1)
         return package_license, homepage
-    return None
+    return None, None
+
+
+@dataclass(frozen=True)
+class PackageInfo:
+    """Represents the package metadata we are extracting."""
+    package: str
+    licence: str
+    category: str
+    homepage: str
+
+    def __lt__(self, other: "PackageInfo") -> bool:
+        return self.licence < other.licence
+
+    def __iter__(self):
+        return iter((self.package, self.licence, self.category, self.homepage))
+
+    @classmethod
```

Comment:
Nit Any reason why this is not a `@staticmethod`? The argument `cls` seems to be unused and this would trigger pylint if we ever decide to lint outside the scaaml directory.

Reply:
Fixed.